src/test/ebl.spec.js

```diff
@@ -3,15 +3,16 @@ import { Selector } from "testcafe";
 
 fixture("Token Document Rendering").page`http://localhost:3000`;
 
-const VerifyDocuments = Selector("[data-testid='navbar-verify-documents']");
-
 test("Token is verified and rendered correctly", async () => {
   await uploadDocument("./fixture/ebl.json");
   await validateIssuerTexts(["TRADETRUST.IO"]);
   await validateIframeTexts(["BILL OF LADING FOR OCEAN TRANSPORT OR MULTIMODAL TRANSPORT"]);
 });
 
-test("Should be able to render certificate twice consecutively", async (t) => {
+// Skipped test due to changing verify to different page
+const VerifyDocuments = Selector("button").withText("Verify Doc");
+// eslint-disable-next-line jest/no-disabled-tests
+test.skip("Should be able to render certificate twice consecutively", async (t) => {
```

Comment:
still need to skip?

Reply:
still need skip, cause don't have the page